client/tracing/src/lib.rs

```diff
@@ -63,8 +62,8 @@ pub enum TracingReceiver {
 	Log,
 	/// Output to telemetry
 	Telemetry,
-	/// Output to Grafana
-	Grafana,
+	/// Output to Prometheus endpoint
```

Comment:
As far as I can tell, there has not been any objections to removing any Prometheus related functionality from the `tracing` crate.

Reply:
That makes sense to me, at least for now.